bukuserver/response.py

```diff
@@ -1,4 +1,40 @@
-response_template = {
-    "success": {'status': 0, 'message': 'success'},
-    "failure": {'status': 1, 'message': 'failure'}
-}
+from enum import Enum
+from flask import jsonify
+from flask_api import status
+
+
+class Status(Enum):
+    OK = (status.HTTP_200_OK, 0)
+    BAD = (status.HTTP_400_BAD_REQUEST, 1)
+    NOT_FOUND = (status.HTTP_404_NOT_FOUND, 1)
+
+
+class Response(Enum):
+    SUCCESS = (Status.OK, "Success.")
+    FAILURE = (Status.BAD, "Failure.")
+    JSON_NOT_VALID = (Status.BAD, "Failed to decode JSON data.")
+    INPUT_NOT_VALID = (Status.BAD, "Request not valid")
+    TAGS_MISSING = (Status.BAD, "Replacement tags are missing.")
+    TAG_NOT_FOUND = (Status.NOT_FOUND, "Tag not found.")
+    TAGS_NOT_LIST = (Status.BAD, "List of tags expected.")
+    TAG_NOT_VALID = (Status.NOT_FOUND, "Invalid tag.")
+
+    def get_status_code(self):
+        return self.value[0].value[0]
+
+    def get_json(self, data :dict = None):
+        json = {'status': self.value[0].value[1], 'message': self.value[1]}
+        if isinstance(data, dict):
+            json.update(data)
+        return json
+
+    def __call__(self, *args, **kwargs):
+        """
+        Kwargs
+           data (dict): updates the response's JSON
+        """
+        return (jsonify(self.get_json(kwargs.get('data'))),
+                self.get_status_code(), {'ContentType': 'application/json'})
+
+
+response_from_flag = lambda flag: Response.SUCCESS() if flag else Response.FAILURE()
```

Comment:
This would probably make more sense as a [`@staticmethod`](https://docs.python.org/3.7/library/functions.html#staticmethod) of `Response` instead, usage wise…